features/topology-map/plugins/org.opennms.features.topology.plugins.topo.graphml/src/main/java/org/opennms/features/topology/plugins/topo/graphml/GraphMLTopologyProvider.java

```diff
@@ -152,6 +162,10 @@ private static int getDefaultSzl(GraphMLGraph graph) {
         return Defaults.DEFAULT_SEMANTIC_ZOOM_LEVEL;
     }
 
+    public GraphMLMetaTopologyProvider getMetaTopologyProvider() {
```

Comment:
Please do not expose the full `MetaTopologyProvider`. We should introduce something new, e.g.
`OppositeVertexAccessor` which is exposed.